kolibri/core/hooks.py

```diff
@@ -0,0 +1,40 @@
+"""
+Kolibri Core hooks
+------------------
+
+WIP! Many applications are supposed to live inside the core namespace to make
+it explicit that they are part of the core.
+
+Do we put all their hooks in one module or should each app have its own hooks
+module?
```

Comment:
The other thing that had seemed odd in discussions in the dev room, is the fact that the architecture for plugins and hooks resides in the plugins module. Because it is part of the core architecture of Kolibri, it seems like all of the abstract base classes for plugins and hooks should actually be in a `core.plugins`(?!) (or something that makes it clear that this is core) namespace.

The `plugins` namespace then is *just* for plugins.